### ISSUE TITLE
VBET: Bring in FCODEs to all Intermediate and Output Feature Classes

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/riverscapes-tools#456]
Related to https://github.com/Riverscapes/RaveAddIn/issues/171#issuecomment-946964728, we need to be able to apply definition queries so we can produce one output for the "entire" network, but filter our analyses and default display. @philipbaileynar and I discussed.

We will have the situation where polygons and centerlines we want to do this for are part of riverscapes that have multiple NHD polylines in them some from perennial and some from non perennial. In these instances, if any thing intersecting it is perennial, make the IsPerrenial 1.

**Comment by joewheaton** [comment]
@philipbaileynar, I forgot about this one in our discussion and #674. How difficult is this? We can also realistically bring this in GNAT. I will not let go of getting FCODE onto IGOs, and would prefer it VBET, but I could live with it in GNAT. Thoughts? Preferences?  @KellyMWhitehead how hard is this one?

**Comment by philipbaileynar** [comment]
This is difficult for reasons to complicated to explain here. FCodes are something that we can recover and associate with metrics in GNAT.

**Comment by joewheaton** [comment]
@jtgilbert I sort of forgot about this one. Let me rephrase it in a simple way. Do we have attributes in VBET that would allow us to identify (maybe by levelpath) the perennial, vs intermittent vs. ephemeral levelpaths? This is important for @shelbysawyer for the summary reports and to say what % of landscape is valley bottom vs. what % of landscape is in perennially flowing riverscape valley bottoms?

**Comment by jtgilbert** [comment]
The stream network has both level path and FCode, so it could be used to join FCode to any other feature class with the levelpath ID (e.g. IGOs DGOs).

**Comment by philipbaileynar** [comment]
@jtgilbert please re-read and closed if solved or no longer relevant.

**Comment by joewheaton** [comment]
Are you confirming this is done? I absolutely still need these. Just confirming....

**Comment by jtgilbert** [comment]
@joewheaton Yeah, the latest runs will have FCode on IGOs and DGOs
